games-action/minetest/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<herd>games</herd>
```

Comment:
> Isn't that worth it to patch the build system(yet again) or is it sufficient to add the dependency?

It is already optional (via `-DENABLE_SPATIAL=<ON/OFF>`). So it could be made either
* a USE flag (with the description "Enable SpatialIndex AreaStore backend")
* just enable it unconditionally